e2e/log.go

```diff
@@ -10,6 +10,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
```

Comment:
I don't think this will remove go lint warning, looks like it says that the comment for a non-main package should start from "Package <package-name>"
See this: https://chromium.googlesource.com/external/github.com/golang/lint/+/refs/heads/master/lint.go#424

Reply:
The comment above the package was license based and not related to the package declaration. Since they were written together, golint was associating them together. The addition of a new line removes that association, which in turn removes the golint warning :)